### ISSUE TITLE
When you are using lora you have to switch lora target models to c_proj

### ISSUE THREAD

**Issue by ktam3** [instructlab/instructlab#1987]
TBD

**Comment by ktam3** [comment]
Copying over @RobotSail chat

> It varies depending on the value used for --is-padding-free
But I think we should have the default configs use the correct modules
I imagine that 80% of customers will just run stuff out of the box without thinking much about it, so the default path needs to be the one that just works
I can make the PR for this after we merge your config PRs
Actually ill just make it and then base on top of yours

**Comment by nathan-weinberg** [comment]
This bug is a little hard to follow :sweat_smile: can we have a little more context? the template fields are good guidance for these

**Comment by RobotSail** [comment]
@nathan-weinberg Basically the default modules we target to become LoRA layers have their names changed when we run with `--is-padding-free=true` so that the only equivalent module becomes `c_proj`. If you run a LoRA train with `--is-padding-free` you can recreate it.

**Comment by nathan-weinberg** [comment]
So is this a duplicate of https://github.com/instructlab/instructlab/issues/1906?

**Comment by RobotSail** [comment]
Yes
